### ISSUE TITLE
How do we know which version of BrAPI is implemented? (and I mean sub-version)

### ISSUE THREAD

**Issue by guignonv** [plantbreeding/API#181]
[OBSOLETE: In the URL we got "/v1/" but how how do we know if it is v1**.1** or v1**.2** or ...?]

I propose to use the root "/" call (not defined yet) to display information about current implementation:
```
{
    "provider": "Musa Germplasm Information System",
    "description": "Explore Banana Diversity",
    "contact": {
        "email": "mgis@crop-diversity.org",
        "www": "https://www.crop-diversity.org/mgis/"
    },
}

```

[EDIT: removed version-related things as it has been discussed and solved at the hackathon]

**Comment by c5sire** [comment]
Indeed - maybe the root call could point to the swagger file for the server?

**Comment by guignonv** [comment]
Another proposition has been made to add versioning to the "call" calls so it could be managed at a call level. It might be useful when you implemented all the calls at v1.2 and only updated a couple of other calls to v1.3 for instance...
But then, what to do with calls that changed name like "allelematrix" into "allelematrices"?

**Comment by BrapiCoordinatorSelby** [comment]
Closing: solved in v1.2
A `versions` array was added to each call response in the /calls call.

If the path of a call changes (ie "allelematrix" into "allelematrices") the old call will be marked Deprecated, but will still be present in the spec, and the new call will be added to the spec. so to support multiple versions, you will need to implement both calls, the new and the deprecated. If you only care about supporting the latest version, you can ignore the deprecated call.

Similarly, if a response object changes, any existing fields which are changed or removed will be marked as Deprecated and new fields will be added as needed. To support multiple versions, simply return all valid and deprecated fields. To only support the latest version, return only the valid fields.
